build.gradle

```diff
@@ -0,0 +1,27 @@
+buildscript {
+	ext {
+		springBootVersion = '1.5.6.RELEASE'
+	}
+	repositories {
+		mavenCentral()
+	}
+	dependencies {
+		classpath("org.springframework.boot:spring-boot-gradle-plugin:${springBootVersion}")
+	}
+}
+
+apply plugin: 'java'
+apply plugin: 'org.springframework.boot'
+
+version = '0.0.1-SNAPSHOT'
+sourceCompatibility = 1.8
+
+repositories {
+	mavenCentral()
```

Comment:
jcenter is used here more than mavenCentral, any reason you've defaulted to mavenCentral?

Reply:
This file was generated by spring initializr - that's how mavenCentral got here. My plan was to make it the initial version and refine, but will change this bit now.